test/test_torch.py

```diff
@@ -11459,7 +11459,10 @@ def test_reduction_empty(self, device):
         self.assertEqual(torch.ones((2, 1, 4), device=device, dtype=torch.uint8), xb.all(1, keepdim=True))
         self.assertEqual(torch.ones((), device=device, dtype=torch.uint8), xb.all())
 
-    def test_addcdiv(self, device):
+    # TODO: add back integral types once integer division is reenabled for addcdiv
+    @dtypesIfCUDA(torch.half, torch.float, torch.double)
+    @dtypes(torch.float, torch.double)
+    def test_addcdiv(self, device, dtype):
```

Comment:
does this belong in the PR?  I didn't see any changes to addcdiv in this change.

Reply:
This change is more correctly part of the sister PR: https://github.com/pytorch/pytorch/pull/34570. I put all the corrections in this one, however. I can move it back.

Reply:
Moved out of this PR.